syntax="proto3";
package com.bitcoin.proto3;

import "OutputV3.proto";

message PaymentDetails {
    string          network       = 1;
    repeated Output outputs       = 2;
    uint64          time          = 3;
    uint64          expires       = 4;
    string          memo          = 5;
    string          payment_url   = 6;
    bytes           merchant_data = 7;

    string          someNewField  = 9;
    int32           someNewInt    = 10;
}
